int16 cid
int16 oid
float32[8] pos
time update_time
bool active